docs/dev/setup/gitlabci-gitlab.rst

```diff
@@ -28,62 +28,60 @@ For a production setup of GitLab, also see the documentation of the GitLab and J
 GitLab
 """"""
 
-1. Depending on your operating system, it is necessary to update the host file of your machine to include the following line:
+1. Depending on your operating system and your docker installation, it is necessary to update the host file of your machine to include the following line:
 
     .. code:: text
 
         127.0.0.1       host.docker.internal
         ::1             host.docker.internal
 
-2. Configure GitLab
+2. Start GitLab and the GitLab Runner
     .. code:: bash
 
-        cp docker/env.example.gitlab-gitlabci.txt docker/.env
+        docker-compose -f docker/gitlab-gitlabci.yml up -d
 
-3. Start GitLab and the GitLab Runner
+3. Get your GitLab root password
     .. code:: bash
 
-        docker-compose -f docker/gitlab-gitlabci.yml --env-file docker/.env up --build -d
+        docker exec artemis-gitlab grep 'Password:' /etc/gitlab/initial_root_password
 
-4. Get your GitLab root password
-    .. code:: bash
-
-        docker exec -it gitlab grep 'Password:' /etc/gitlab/initial_root_password
+4. Generate an access token
+    Go to ``http://localhost:8081/-/profile/personal_access_tokens`` and generate an access token with all scopes.
+    This token is used for generating a runner token and in the Artemis configuration as ``artemis.version-control.token``.
 
-5. Generate an access token
-    Go to ``http://host.docker.internal/-/profile/personal_access_tokens`` and generate an access token with all scopes.
-    This token is used in the Artemis configuration as ``artemis.version-control.token``.
-
-6. Allow outbound requests to local network
+5. Allow outbound requests to local network
     For setting up the webhook between Artemis and GitLab, it is necessary to allow requests to the local network.
-    Go to ``http://host.docker.internal/admin/application_settings/network`` and allow the outbound requests.
+    Go to ``http://localhost:8081/admin/application_settings/network`` and allow the outbound requests.
     More information about this aspect can be found in the `GitLab setup instructions <#gitlab-access-token>`__ (step 12).
 
 GitLab Runner
 """""""""""""
 
-1. Register a new runner
-    Login to your GitLab instance and open ``http://host.docker.internal/admin/runners``.
-    Click on ``Register an instance runner`` and copy the registration token.
+1. Create a runner token
+    In order to create a token which can be used to register a runner, execute the command below with the personal access token generated before.
 
+    .. code:: bash
```

Comment:
Is there a specific reason why the runner token is now generated via a command rather than the UI?
I assume especially Windows users might not have `curl` easily available.

Reply:
Even in more advanced systems, generating personal access tokens, etc., is done just once, whereas there might be multiple runners, each with their individual runner token. Thus, for generating Runner tokens, there might be the need to do a bulk deployment, for which doing this via the command line might be helpful.
I am currently figuring out how to include this in the Ansible Collection.

Nevertheless, thanks for the heads-up about the Windows users. I also added a reference to the UI.